templates/block/block--share-this-page-block.html.twig

```diff
@@ -36,12 +36,18 @@
   'has_constrain': false,
   'hide_content_wrapper': true
 } %}
+
   {% block content %}
-    {% include '@components/social-share/social-share.twig' with {
-      'facebook': content['links']['#facebook'] ? content['links']['#facebook']['url'] : null,
-      'twitter': content['links']['#twitter'] ? content['links']['#twitter']['url'] : null,
-      'linkedin': content['links']['#linkedin'] ? content['links']['#linkedin']['url'] : null,
-      'email': content['links']['#email'] ? content['links']['#email']['url'] : null,
-    } %}
+    {# Hide Social Media Share Icons if selected in the theme settings #}
+    {% if hide_social_media_share_icons == "0" %}
```

Comment:
Should we set this to `!== "1"`, just to be sure a site with no value set still displays the social media icons?

Reply:
I'm pretty sure it's right as is but I'll double check in the morning

Reply:
OK, after looking at `theme_get_setting` more closely, I see that it returns: The value of the requested setting, NULL if the setting does not exist.

I'd be comfortable if we changed `{% if hide_social_media_share_icons == "0" %}` to `{% if hide_social_media_share_icons == 0 %}` because `0 == NULL` will evaluate to TRUE, but `"0" == NULL` will not. That way, if a site is returning NULL for whatever reason, the social icons will still display by default.